Eliminate codegen from behaviors in alica_tests

triccyx opened the pull request:
Working on alica_tests behaviours:
- Remove code generation TAGS in behaviours
- Add dynamic loading feature to behaviours

Commit:
first experiment

Dale-Koenig commented:
@triccyx You requested review but it is still marked draft.  What is the status of the PR?

triccyx committed:
Merge branch 'devel' into refactory-behaviour-in-tests

Commit:
removed cod gen tag and added dynamic loading to behaviours

Commit:
Merge branch 'refactory-behaviour-in-tests' of https://github.com/rapyuta-robotics/alica into refactory-behaviour-in-tests

Commit:
step 002

triccyx committed:
Merge branch 'devel' into refactory-behaviour-in-tests

triccyx commented:
@Dale-Koenig for this PR I have removed code gen TAGs and added dynamic loading method.
The library can be done as soon as all other parts (plans, conditions) are modified.

triccyx committed:
Merge branch 'devel' into refactory-behaviour-in-tests

ntfshard commented on supplementary/alica_ros1/alica_tests/src/test/Expr/src/ConditionCreator.cpp:
```diff
@@ -6,26 +6,6 @@
-#include <alica_tests/Behaviour/TestParameterPassingBehaviour831400441334251602.h>
-#include <alica_tests/Behaviour/TriggerA1428508297492.h>
-#include <alica_tests/Behaviour/TriggerB1428508316905.h>
-#include <alica_tests/Behaviour/TriggerC1428508355209.h>
```
I guess we could use [Include What You Use](https://github.com/include-what-you-use/include-what-you-use) at the end for make it tidy

triccyx replied:
Interesting ... I will take a look at it!

triccyx committed:
Merge branch 'devel' into refactory-behaviour-in-tests

david-simoes-93 reviewed (approved):
:see_no_evil:
